src/app/[locale]/(default)/events/[id]/loading.tsx

```diff
@@ -0,0 +1,27 @@
+import { Separator } from '@/components/ui/Separator';
+import { Skeleton } from '@/components/ui/Skeleton';
+import { CalendarIcon, MapPinIcon } from 'lucide-react';
+
+export default function EventDetailsLoading() {
+  return (
+    <>
+      <Skeleton className='my-4 h-12 w-3/4 rounded-lg' />
```

Comment:
```suggestion
      <Skeleton className='mt-6 mb-4 h-12 w-3/4 rounded-lg' />
```
To keep same layout as in the actual page